### DISCUSSION TITLE
Free or Free Trial?

### DISCUSSION THREAD

**Discussion by MrAmericanMike** [community/community#147624] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

So today I see this on Github.

![image](https://github.com/user-attachments/assets/e3b41b96-f2bb-4e4a-9802-7593aa59167a)

But then when clicking to start using it, it says it's a free trial. Way different than being free.

It first says: "Copilot Free is active for your account" But then the button says "Start a free trial"

So maybe I am confused, is it Free or just a Free trial?

**Comment by Akash1134** [comment, marked as answer]
Hi @MrAmericanMike , 

Copilot Free is not a trial product. Once you have exhausted its limits, you can start a trial of Copilot Pro. Today, Copilot Free has the following limitations:

- Code completions are limited to 2000 completions per month.
- Copilot Chat is limited to 50 chat messages per month. This limit includes both standard chats and multi-file editing chats in VS Code and Visual Studio.

When you reach these limits, you can upgrade to Copilot Pro to continue using Copilot.

- If you use Copilot Free in VS Code or on GitHub.com and reach the usage limits, you'll receive a notification. The notification includes the reset date for your limits and a link to set up a 30-day free trial of Copilot Pro. After the trial ends, you'll need a paid subscription to keep using Copilot.

- If you use Copilot Free in a different IDE and reach the limits, an error message will appear in your editor. To continue, you can start a 30-day free trial of Copilot Pro in your GitHub account settings. See [Setting up GitHub Copilot for yourself](https://docs.github.com/en/copilot/setting-up-github-copilot/setting-up-github-copilot-for-yourself#1-get-access-to-github-copilot). 

Hope that helps!

**Reply by jonnygehlot** [reply]
Hi, This morning i saved my card details for copilot pro subscription and started my free trail it does not do any payment as 1 month is free trail but after few mintues it says your free trail has ended and when i retry saving the card details it repeats the same thing ....... can you tell me what this issue is ?

**Reply by hananindyyy** [reply]
I have the same issue as jonnygehlot. On the billing & licensing page, my copilot is still in free mode, not the 30 day free trial of copilot pro. I have filled out the payment method & I'm afraid I will be charged for copilot pro next month.

**Reply by Betelsec** [reply]
im having the same issue as well 
so exhausted !

**Reply by xxristoskk** [reply]
I also ran into this issue today. I signed up for the free 30-days, and I got the "upgrade to pro" warning 2 days later.

**Reply by kristianaas5** [reply]
The exact same happened to me. I exhausted my Copilot free and sighed for Copilot Pro, however a minute after there was a mistake and it wanted me to pay now.

**Reply by Andydo21** [reply]
i have the same issue

**Reply by jarmen423** [reply]
I signed up for the free trial and then never got to use it just went striaght back to co pilot free mode with no pro trial

**Reply by Bruzzlee** [reply]
I just had the same experience. I was able to resolve the issue. I used a prepaid credit card as my payment method, which could not be verified. After entering a postpaid credit card, the free trial reappeared and could be activated.

**Comment by JayeshSteed26** [comment]
To be honest after using the copilot pro and its models for more than 10 days i don't really find it that helpful as i thought it was. Small tasks it does great quick but in larger tasks they always leave something out. It feels good to see 300 lines worth of code generated in 10 seconds but mostly you wouldn't have needed that much code. We have to explain too much that with less clarity we could code its with less time than it takes in fixing processing, refixing reprocessing and the End Result ? 

1. Eventually you start doubting your own abilities. 
2. You get addicted to someone else doing your job.
3. In my case i start to forget what i used to know and could write flawlessly.

Just my inner thoughts, others may disagree. 
![SchoolWorkAngryGIF](https://github.com/user-attachments/assets/02f6133f-82c5-4453-a39f-30a071190f72)

**Comment by jonnygehlot** [comment]
Yeah, as you pass more time you'll see more of these, my trial ended in
just 2 weeks, it took 1 week to get it started properly and after closing
my trial falsely when i raised a ticket to resolve it they haven't replied
to it for last 2 weeks.

They are running a scam in the name of free trail, collecting card details.

**Comment by m4sh-wacker** [comment]
:P
